test/integration/amqp/events/events.spec.js

```diff
@@ -0,0 +1,54 @@
+const path = require("path");
+const Promise = require("bluebird");
+const { exec, callIn } = require("../util");
+
+const commonString = " received the event.";
+
+const findResponse = (arr) => arr
+	.map(string => {
+		const line = string
+			.split("\n")
+			.find(a => a.indexOf(commonString) > 0);
+		return line ? line.slice(0, line.indexOf(commonString)) : false;
+	})
+	.filter(a => a);
+
+xdescribe("Test AMQPTransporter events", () => {
```

Comment:
`xdescribe` is the same as `describe.skip`, I did this for the integration tests I wrote so the describe block won't run. These tests require an AMQP server to be running at `amqp://guest:guest@localhost:5672`, so I don't think they would pass in the current CI environment.